### DISCUSSION TITLE
Return Path problem

### DISCUSSION THREAD

**Discussion by ugurtam** [postalserver/postal#2705] [Help with using Postal]
In my case, I need to remove the default return path of Postal, and non-delivery emails should return to original sender. Is this possible by making a change on the codes ?? I need this urgently ! Can anybody help ? I saw some discussions about this, but none of codes in these entries work.

The following method for example, doesn't work
https://github.com/templateria/postal/commit/d5941a99ffbfe7c49bcbeb4aea6ac05d2e1b9c42

**Comment by CarlosFongo** [comment]
Hi, did you find a solution?
